src/main/java/org/spongepowered/api/text/Texts.java

```diff
@@ -233,6 +290,62 @@ public static Text of(Object... objects) {
             }
         }
 
+        if (builder.children.size() == 1) {
```

Comment:
@ST-DDT This change has been merged: https://github.com/SpongePowered/SpongeAPI/commit/c452da63263cb7b948d4235fced67422ddea954a